package.json

```diff
@@ -42,12 +46,16 @@
 		"@babel/core": "^7.9.0",
 		"@babel/plugin-syntax-dynamic-import": "^7.8.3",
 		"@babel/preset-env": "^7.9.5",
+		"@nextcloud/browserslist-config": "^1.0.0",
+		"@nextcloud/eslint-config": "^2.0.0",
+		"@nextcloud/eslint-plugin": "^1.4.0",
 		"@vue/test-utils": "^1.0.0-beta.33",
 		"babel-eslint": "^10.1.0",
 		"babel-jest": "^25.3.0",
 		"babel-loader": "^8.1.0",
 		"css-loader": "^3.5.2",
 		"eslint": "^5.16.0",
+		"eslint-config-nextcloud": "^0.1.0",
```

Comment:
`@nextcloud/eslint-config` is imported so I think this package is no longer necessary.

Reply:
Fixed